src/main.cpp

```diff
@@ -569,8 +573,63 @@ int main(int argc, char *argv[])
         }
     }
 
+    if (c == 'h') 
+    {
+         printf("Usage: wf-recorder [OPTION]... [FILE]...
+Screen recording of wlroots-based compositors
+
+With no FILE, start recording the current screen.
+
+  -a, --audio               Starts recording the screen with audio
+                
+  -c, --codec               Specifies the codec of the video. Supports  GIF output also.
+                            To modify codec parameters, use -p <option_name>=<option_value>
+                
+  -d, --device              Selects the device to use when encoding the video
+                            Some drivers report support for rgb0 data for vaapi input but really only support yuv.
+                            Use the -t or --to-yuv option in addition to the vaapi options to convert the data in software,
+                            before sending it to the gpu.
+                
+  -g, --geometry            Selects a specific part of the screen.
+  
+  -h, --help                Prints this help screen.
+                
+  -l, --log                 Generates a log on the current terminal. Debug purposes. 
+                
+  -o, --output              Specify the output where the video is to be recorded.
+                
+  -p, --codec-param         Change the codec parameters.
+                            -p <option_name>=<option_value>
+
+  -t, to-yuv                Use the -t or --to-yuv option in addition to the vaapi options to convert the data in software,
```

Comment:
Maybe leave this out from this PR until we merge #32, then me or @soreau will add the docs for `-t` and `-e`

Reply:
@ammen99 #30 was merged so -t already exists.

Reply:
Didn't add the -e option